xdsl/transforms/experimental/dmp/stencil_global_to_local.py

```diff
@@ -636,39 +642,40 @@ class DmpDecompositionPass(ModulePass, ABC):
     Represents a pass that takes a strategy as input
     """
 
+
+@dataclass
+class DistributeStencilPass(DmpDecompositionPass):
```

Comment:
I know that we originally have this concept of "distributing a stencil", but maybe we should be smoothly starting to think on how to use the fact that we decompose the grid, not the stencil....
so like at some point to move to a decomposeGrid or sth similar Pass

Reply:
Yes! But I would wait with that for version 2 of the dmp dialect. It's definitely an interesting topic though!